src/js/components/Select/index.d.ts

```diff
@@ -26,17 +33,17 @@ export interface SelectProps {
   dropTarget?: object;
   dropProps?: DropProps;
   focusIndicator?: boolean;
-  icon?: boolean | ((...args: any[]) => any) | React.ReactNode;
+  icon?: boolean | (() => React.ReactElement) | React.ReactNode;
```

Comment:
Does this actually works without the `args`?

Reply:
yes

Reply:
Please see this line: https://github.com/grommet/grommet/blob/f4003357390dd83e250c535f90c20e19368043da/src/js/components/Select/Select.js#L287 The type definition should include those 2 props (color and size). I've actually been working on some changes to the Select component and this is something I noticed.